pills/05-functions-and-imports.xml

```diff
@@ -1,8 +1,348 @@
 <chapter xmlns="http://docbook.org/ns/docbook"
-      xmlns:xlink="http://www.w3.org/1999/xlink"
-      xmlns:xi="http://www.w3.org/2001/XInclude"
-      version="5.0"
-      xml:id="functions-and-imports">
+xmlns:xlink="http://www.w3.org/1999/xlink"
+xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0"
+xml:id="functions-and-imports">
+
+      <title>functions and imports</title>
+
+<section>
+      <title>Functions</title>
+      <para>
```

Comment:
these introductory para's don't need a section just on their own, just "leading paragraphs" or whatever docbook wants to call them.